src/Text/Taggy/Combinators.hs

```diff
@@ -11,7 +11,6 @@
 module Text.Taggy.Combinators (hasName, hasAttr, getAttr, innerText, (//), (/&), (/*), trees, subtrees) where
 
 import Prelude hiding (lookup)
-import Data.Monoid (mconcat)
```

Comment:
Oh, sorry, I wasn't thinking about backwards compatibility. Yeah in 7.10 it's still in `Data.Monoid`, but it's a warning since it's implicitly imported from `Prelude`.

There's a few more of these deletions: 
https://github.com/alpmestan/taggy/pull/11/files#diff-4deaf6385b35b0b5596d24d68210e346L13 
https://github.com/alpmestan/taggy/pull/11/files#diff-4eeaa834414e0613272a7a0124d674ebL6 
https://github.com/alpmestan/taggy/pull/11/files#diff-5eccf2d5802bbe967d7e4a6895c054b4L5

They are just warnings, so ultimately you could just ignore this PR entirely.

But I'm fine with adding CPP conditionals.